### PR TITLE
Bemenu selection for :open and :open -t

### PR DESCRIPTION
**Pull request by halfwit**

### CODE REVIEW

**Comment by The-Compiler** [comment]
Sorry for not getting to this (or the older PR) earlier!

I don't see the need for two scripts - you can simply do `:bind o spawn --userscript /path/to/userscripts/qutedmenu open` and get rid of the second script :wink:

Can you remove the other one and adjust the comment accordingly? Note you can just push and the PR will update, no need to open a new one.

I'll also leave a line comment with a little nitpick - I usually don't do that for userscripts, but here I really have to :wink:

File: misc/userscripts/qutedmenu [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    done < "$confdir"/qutebrowser/bookmarks/urls
+
+    # Finally history
+    while read -r _ url; do
+        printf -- '%s\n' "$url"
+    done < "$datadir"/qutebrowser/history
+    }
+
+get_selection() {
+    opts+=(-p Qutebrowser)
```
Comment (The-Compiler) [inline misc/userscripts/qutedmenu]: Please make this `qutebrowser` - `Qutebrowser` looks so wrong to me. :laughing:

Comment (halfwit) [inline misc/userscripts/qutedmenu]: Will do!

**Comment by halfwit** [comment]
Updated.

**Comment by The-Compiler** [comment]
The comment still says "Broken out to two files to handle flag passing", but I'll just remove this line myself.

Thank you!

**Pull request merged**